src/Twig/Extensions.php

```diff
@@ -37,7 +40,7 @@ class Extensions extends \Twig_Extension
      */
     public function __construct(KernelInterface $kernel, MaintenanceMode $maintenanceMode)
     {
-        $this->kernelRootDir = $kernel->getRootDir();
+        $this->kernelRootDir = $kernel->getProjectDir();
```

Comment:
Rootdir is not the same as ProjectDir, so dataland breaks.

Reply:
also new search is throwing a 500 now..looking into it

Reply:
Adding ` . '/src';` would give the same result.

Reply:
https://pastebin.com/fHjwy0ju  
Looks like our LogActionItemEvent isn't worknig as an Eventdispatcher Event.

Reply:
That's just a type hinting issue. It's parent must be updated too.

Reply:
It was more than that, the dispatch function signature changed.